src/lwc/gePaymentGatewayManagement/gePaymentGatewayManagement.html

```diff
@@ -0,0 +1,82 @@
+  ~       its contributors may be used to endorse or promote products derived
+  ~       from this software without specific prior written permission.
+  ~
+  ~     THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+  ~     "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+  ~     LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+  ~     FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+  ~     COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+  ~     INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+  ~     BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+  ~     LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+  ~     CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+  ~     LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+  ~     ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+  ~     POSSIBILITY OF SUCH DAMAGE.
+  -->
+
+<template>
+    <template if:true={isSuccess}>
+        <c-util-alert-banner theme="success"
+                             icon="utility:success"
+                             size="small"
+                             message="Gateway updated.">
+        </c-util-alert-banner>
+    </template>
+    <template if:true={isError}>
+        <c-util-alert-banner theme="failure"
+                             icon="utility:error"
+                             size="small"
+                             message={errorMessage}>
+        </c-util-alert-banner>
+    </template>
+    <lightning-layout multiple-rows="true">
+        <div if:true={showSpinner}
+             class='spinner-container'>
+            <lightning-spinner alternative-text={CUSTOM_LABELS.messageLoading}
+                               size='large'>
+            </lightning-spinner>
+            <p class='loading-text slds-text-heading_medium'>{loadingText}</p>
+        </div>
+        <lightning-layout-item size="12">
+            <h1 class="slds-text-heading_medium slds-p-top-medium">Elevate Gateway Management</h1>
+        </lightning-layout-item>
+        <lightning-layout-item size="12">
+            <p class="slds-text-body_regular slds-p-vertical_medium">NPSP uses the gateway that was added to Elevate most
+                recently. To override this, click Edit and enter the gateway ID for the one you'd like to use instead.
+                Find the gateway ID in the Elevate Admin Console | Gateway Management.</p>
+        </lightning-layout-item>
+        <template if:true={isReadState}>
+            <lightning-layout-item size="12" class="slds-text-align_center">
+                <lightning-button label="Edit" data-id="editGatewayButton" onclick={handleEdit}></lightning-button>
+            </lightning-layout-item>
+        </template>
+        <lightning-layout-item size="4">
+            <template if:true={isReadState}>
```

Comment:
Can these two templates that use the same if conditional (`template if:true={isReadState}`) be combined into a single template?

Reply:
@kenlewis-sfdo Pushed an update to use a single isReadOnly getter so now we're using if:true to render the edit button and if:false to render the Save/Cancel button group.